shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/parameter/impl/EncryptInsertOnDuplicateKeyUpdateValueParameterRewriter.java

```diff
@@ -55,15 +59,16 @@ public void rewrite(final ParameterBuilder parameterBuilder, final InsertStateme
             int columnIndex = index;
             String encryptLogicColumnName = onDuplicateKeyUpdateValueContext.getColumn(columnIndex).getIdentifier().getValue();
             Optional<EncryptAlgorithm> encryptorOptional = getEncryptRule().findEncryptor(tableName, encryptLogicColumnName);
+            Map<String, String> encryptContextMap = new EncryptContext(insertStatementContext.getSchemaName(), tableName, encryptLogicColumnName).of();
```

Comment:
@cheese8 Can we consider that the logic of encryption and decryption parameters is implemented by the user in the form of SPI?